src/Microsoft.AspNet.OData.Shared/Query/SelectExpandQueryOption.cs

```diff
@@ -88,10 +87,11 @@ internal SelectExpandQueryOption(string select, string expand, ODataQueryContext
                 throw Error.Argument(SRResources.SelectExpandEmptyOrNull);
             }
 
-            IEdmEntityType entityType = context.ElementType as IEdmEntityType;
-            if (entityType == null)
+            IEdmStructuredType structuredType = context.ElementType as IEdmStructuredType;
+
+            if (structuredType == null)
```

Comment:
>structuredType == null [](start = 16, length = 22)

You don't ever use structuredType; why not remove ln 90 and just do if (!(context.ElementType is IEdmStructuredType)) as you did in ln 55?